src/Microsoft.IdentityModel.Tokens/TokenValidationParameters.cs

```diff
@@ -80,6 +80,15 @@ namespace Microsoft.IdentityModel.Tokens
     public delegate bool LifetimeValidator(DateTime? notBefore, DateTime? expires, SecurityToken securityToken, TokenValidationParameters validationParameters);
 
     /// <summary>
+    /// Definition for TokenReplayValidator.
+    /// </summary>
+    /// <param name="securityToken">The <see cref="SecurityToken"/> being validated.</param>
+    /// <param name="expirationTime">The 'expiration' time found in the <see cref="SecurityToken"/>.</param>
+    /// <param name="validationParameters"><see cref="TokenValidationParameters"/> required for validation.</param>
+    /// <returns></returns>
+    public delegate bool TokenReplayValidator(string securityToken, DateTime? expirationTime, TokenValidationParameters validationParameters);
```

Comment:
Could you change the order of the parameters to match LifetimeValidator? securityToken after expirationTime.